django_project/frontend/src/components/Footer/index.jsx

```diff
@@ -17,7 +17,7 @@
    NAVBAR
    ========================================================================== */
 
-import React from "react";
```

Comment:
Hi @prishashethh
Could create new component instead using this Footer?
This is needed on the project